src/energyplus/ForwardTranslator/ForwardTranslateThermalZone.cpp

```diff
@@ -946,6 +945,25 @@ namespace energyplus {
         }    // if dsoa
       }      // loop on spaces
 
+      if (!atLeastOneDSOAWasWritten && sizingZoneIdf) {
+        // Controller:MechnicalVentilation: Design Specification Outdoor Air Object Name <x>
+        // > If this field is blank, the corresponding DesignSpecification:OutdoorAir object for the zone will come from
+        // > the DesignSpecification:OutdoorAir object referenced by the Sizing:Zone object for the same zone.
+        // > ***If no such zone match is found, default values from the IDD will be used for the DesignSpecification:OutdoorAir object
+        // > which is 0.0094 m3/s-person.***
+        // Apparently I need to set an empty one or something
+        auto& dsoaObject = m_idfObjects.emplace_back(IddObjectType::DesignSpecification_OutdoorAir);
+
+        dsoaObject.setString(DesignSpecification_OutdoorAirFields::Name, modelObject.nameString() + " Zero air DSOA");
+        dsoaObject.setString(DesignSpecification_OutdoorAirFields::OutdoorAirMethod, "Maximum");
+        dsoaObject.setDouble(DesignSpecification_OutdoorAirFields::OutdoorAirFlowperPerson, 0.0);
+        dsoaObject.setDouble(DesignSpecification_OutdoorAirFields::OutdoorAirFlowperZoneFloorArea, 0.0);
+        dsoaObject.setDouble(DesignSpecification_OutdoorAirFields::OutdoorAirFlowperZone, 0.0);
+        dsoaObject.setDouble(DesignSpecification_OutdoorAirFields::OutdoorAirFlowAirChangesperHour, 0.0);
+
+        sizingZoneIdf->setString(Sizing_ZoneFields::DesignSpecificationOutdoorAirObjectName, dsoaObject.nameString());
+      }
```

Comment:
Write an empty DSOA. ThermalZone::combineSpaces was actually doing this for you already... because it creates an DSOA for the Space with all zeroes.